deepchem/utils/data_utils.py

```diff
@@ -143,6 +143,36 @@ def unzip_file(file: str,
     zip_ref.extractall(dest_dir)
 
 
+class UniversalNamedTemporaryFile:
+  """The implementation for cross platform NamedTemporaryFile.
+
+  `tempfile.NamedTemporaryFile` causes a permission error on Windows.
+  This implementation avoids the error, please see threads on the stackoverflow [1]_.
+
+  References
+  ----------
+  .. [1] https://stackoverflow.com/questions/23212435/permission-denied-to-write-to-my-temporary-file
```

Comment:
I looked up this URL.  Someone did post this code, but other better solutions have a lot more votes.  According to the top voted answer, all you need to do is add the option `mode='w'` to the NamedTemporaryFile constructor?

Reply:
> all you need to do is add the option mode='w' to the NamedTemporaryFile constructor?

No. The line that happens error is here
https://github.com/deepchem/deepchem/blob/6c5a5405acea333ee7a65a798ddb5c9df702a0b8/deepchem/data/data_loader.py#L432
This line has already received `mode='w'` option.